tests/phpunit/integration/tests/REST_API/Embed_Controller.php

```diff
@@ -253,7 +239,7 @@ public function test_removes_trailing_slashes() {
 
 	public function test_local_url() {
 		wp_set_current_user( self::$editor );
-
+		$this->set_permalink_structure( '' );
```

Comment:
It is interesting that this test fails without this here. Might be something to look into.

Reply:
I'll take a look at this in #8513